libs/wingsdk/src/platform/parameter-registrar.ts

```diff
@@ -0,0 +1,163 @@
+import Ajv from "ajv";
+import { Construct } from "constructs";
+import { loadPlatformSpecificValues } from "./util";
+
+/**
+ * Parameter Registrar
+ *
+ * This class is used to register and lookup parameter values.
+ */
+export class ParameterRegistrar extends Construct {
```

Comment:
Does this need to be a construct? Maybe it's possible to avoid it...

If it does let's at least make hide it in the Wing console, I think there's an API like this available
```
std.Node.of(this).hidden = true;
```

Reply:
Im leveraging the fact that it exists as a construct to utilize preSynth call to validate all schemas against the input values. I think it makes it easier to aggregate all schemas, and then run an `allOf` validation to get a full list of missing parameters, and return that to users.